functions/hookup.php

```diff
@@ -93,7 +93,7 @@ public function load_hookup($topic_id)
 		$sql = 'SELECT date_id, date_time, text
 				FROM ' . $this->hookup_dates_table . '
 				WHERE topic_id=' . (int) $topic_id . '
-				ORDER BY date_time ASC';
+				ORDER BY date_time ASC, date_id';
```

Comment:
Can you please also add an "ASC" or "DESC" so it is clear in what order they should appear? 

Actually, we may want to sort them alphabetically instead of sorting them by ID, which again is kind of random.